js/product-details.js

```diff
@@ -1,161 +1,282 @@
+    container.appendChild(textrow);
   }
-  const productValuerow = document.getElementsByClassName(
-    "product-details-value"
-  )[0].children;
-  for (i = 0; i < productValuerow.length; i++) {
-    productValuerow[i].classList.add("product-details-value--row--none");
+
+  const addPlacementClassesToElement = function (elem, placing) {
+    elem.classList.add("product-details-value--" + placing);
+    elem.classList.add("product-details-value--" + placing + "--none");
+    elem.classList.add("product-details-value--" + placing + "--" + key);
+    return elem;
   }
-};
 
-const windowWidth = function () {
-  return document.getElementsByClassName("content-width")[0].offsetWidth;
-};
\ No newline at end of file
+  const addClickEventListenerToButtons = function () {
+    let productDetails = document.getElementsByClassName("details-button");
+    productDetails = [].map.call(productDetails, function (item) {
+      item.addEventListener("click", function () {
+        const SIZE_FOR_PRODUCT_DETAILS_IN_COLUMN = 769;
+        if (windowWidth() > SIZE_FOR_PRODUCT_DETAILS_IN_COLUMN) {
+          removeClass();
+          this.classList.add("details-button-active--row");
+          let valuecontainer = document
+            .getElementsByClassName(
+              "product-details-value--row--" + this.innerHTML
+            )[0]
+            .classList.toggle("product-details-value--row--none");
+        } else {
+          this.classList.toggle("details-button-active--column");
+          document
+            .getElementsByClassName(
+              "product-details-value--column--" + this.innerHTML
+            )[0]
+            .classList.toggle("product-details-value--column--none");
+        }
+      });
+    });
+  };
+
+  const removeClass = function () {
+    const productDetails = document.getElementsByClassName("product-details")[0]
+      .children;
+    for (i = 0; i < productDetails.length; i++) {
+      productDetails[i].classList.remove("details-button-active--row");
+    }
+    const productValuerow = document.getElementsByClassName(
+      "product-details-value"
+    )[0].children;
+    for (i = 0; i < productValuerow.length; i++) {
+      productValuerow[i].classList.add("product-details-value--row--none");
+    }
+  };
+
+  const windowWidth = function () {
+    return document.getElementsByClassName("content-width")[0].offsetWidth;
+  };
+})();
+
+const saveComment = function (position) {
+  let commentText = document.getElementsByClassName("product-details-comment-input--" + position)[0].value;
+  productId = location.search.substr(1).split("_")[1];
+
+  let time = new Date;
+
+  time = time.getDate() + "/" + (time.getMonth() + 1) + "/" + time.getFullYear() + " " + time.getMinutes() + ":" + time.getHours();
+  let coment = {};
+  coment[productId] = [{
+    "name": logInfo[1],
+    "time": time,
+    "comment": commentText
+  }];
+  alert("postfunction " + JSON.stringify(coment));
```

Comment:
whaaaat?) alert?)

Reply:
this is the type of reminder that there should be a function, that will send comments to the server))

Reply:
for reminder use TODO
it's a construction which is recognized by IDE and you could at any time look throw all your TODOs

to make it write like comment and '@' before TODO
// @TODO something

you will see it will be even a different color